model.py

```diff
@@ -721,8 +705,19 @@ def __init__(self):
         self._test_dataloader = None
 
         # init multiple gpus context
-        self._place = fluid.CUDAPlace(distributed.Env().dev_id) \
-                    if distributed.Env().nranks > 1 else fluid.CUDAPlace(0)
+        self._place = fluid.CUDAPlace(ParallelEnv().dev_id) \
```

Comment:
Should we add CPUPlace here
Additionally should we need a way to switch between CPU and GPU

Reply:
updated, thanks.